docs/site/tutorials/todo/todo-tutorial-geocoding-service.md

```diff
@@ -225,8 +225,15 @@ export class TodoController {
     todo: Omit<Todo, 'id'>,
   ): Promise<Todo> {
     if (todo.remindAtAddress) {
-      // TODO handle "address not found"
       const geo = await this.geoService.geocode(todo.remindAtAddress);
+
+      if (!geo[0]) {
+        // address not found
+        throw new HttpErrors.BadRequest(
+          `Address "${todo.remindAtAddress}" not found`,
```

Comment:
Nitpick (feel free to ignore).

What happens when the address contains double quote or special characters like a new-line (`\n`)? I think the following message may be better:

```suggestion
          `Address not found: ${todo.remindAtAddress}`,
```